scripts/referral.md

```diff
@@ -0,0 +1,14 @@
+# Referrals
+
+## Hypothesis
+
+The service can go viral based on organic referrals. People will sign up if the value hypothesis is stated by someone they know.
+
+## Metric
+
+Number of new signups with a referral code should increase.
+
+## Script
+
+- Happy holidays! They say people "don't do politics." Let's prove them wrong.
```

Comment:
Is this too Amercianised? I agree we want to be cautious about assumptions around which religious festivals people celebrate.Just a wonder.

I think this isn't human enough and would suggest we lead with a standard: "Hi this is Alex from Movement. Something something. What happens next. What the value is. Etc.

Reply:
For sure, this was a straight copy/paste/replace from the Momentum spam message. Thanks for reminding of the design checklist, I'm very much in holiday mood!